CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 *[CalVer, YY.month.patch](https://calver.org/)*
 
+## 22.7.4
```

Comment:
If you've got multiple PRs open, might be easier to write this under a `## Future` heading and leave `__version__` alone, and I'll handle those after merging (and any merge conflicts).

[Note to self: also pull in [my usual ordering and consistent-increment checks](https://github.com/Zac-HD/hypothesis-jsonschema/blob/afbb2a919aceb8b784d532555946c6313e5be643/tests/test_version.py#L39-L55)]

Reply:
Makes sense :+1: